woocommerce/woocommerce
# Add Payoneer, zipco payment gateways and update Klarna available countries

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/team-ghidorah/issues/181.

P2: pdibGW-1wE-p2#comment-1179

Update the onboarding default payment gateways:

- Add `US/CA/AU/NZ/MX` to Klarna
- Add Payoneer
- Add zipco




### How to test the changes in this Pull Request:




1. Go to OBW and select US as store country
2. Complete OBW with WCPay installed
3. Go to `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`
4. Uncheck `Display suggestions within WooCommerce`
5. Go to `WooCommerce > Home > Payment` task
6. Set up the WCPay 
7. Go to `WooCommerce > Home > AdditionalPayments` task
8. Observe that `Klarna` and `zipco` are shown as Additional Payment options
![Screenshot 2023-03-21 at 14 29 25](https://user-images.githubusercontent.com/4344253/226534769-10fc949b-8cc5-4c7e-b26e-e02f090dae6e.png)
9. Click on zipco's `Get started`
10. Confirm zipco is installed and activated
11. Go to `WooCommerce > Settings`
12. Change store country to `Hong Kong` or `China`
13. Go to the payment task
14. Observe that `Payoneer Checkout` is shown as Additional Payment options
15. Click on Payoneer `Get started`
16. Confirm Payoneer is installed and activated

![Screenshot 2023-03-21 at 14 31 43](https://user-images.githubusercontent.com/4344253/226534785-9c8a5862-13ca-4b52-a1b9-2d552deeedab.png)

19. Optionally, change store country to `CA/AU/NZ/MX` and confirm Klarna` is shown as Additional Payment option



### Other information:




### FOR PR REVIEWER ONLY:
